nextflow.config

```
@@ -9,10 +9,9 @@
 params {
 
   // Workflow flags
-  // TODO nf-core: Specify your pipeline's command line flags
-  genome = false
-  reads = "data/*{1,2}.fastq.gz"
-  singleEnd = false
+  input = null
+  params.chr = null
+  params.no_read_QC = false //By default: QC is perfored on extrcted reads
```

```suggestion
  no_read_QC = false //By default: QC is perfored on extrcted reads
```
params is already in the clause above